src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,25 @@
+package calculator;
+
+import java.util.List;
+
+public class Calculator {
+
+    private static final int RESULT_WHEN_EMPTY = 0;
+
+    private Calculator() {
+    }
+
+    public static int sum(List<Integer> numbers) {
+        numbers.forEach(Calculator::validate);
+
+        return numbers.stream()
+                .reduce(Integer::sum)
+                .orElse(RESULT_WHEN_EMPTY);
```

Comment:
```suggestion
        return numbers.stream()
                .reduce(0, Integer::sum);
```
다음과 같이 코드를 줄여볼수도 있습니다~!

Reply:
이렇게 되면 "빈 값을 입력했을 때 0이 아닌 다른 값을 반환해야 한다"라고 하면 코드를 다시 위와 같이 바뀌어야 할 것 같아요.
그렇다면 비었을 때의 값을 디폴트 값인 0으로 하고 있을 때 더하는 형식으로 해도 되지 않을까요?

Reply:
아하, RESULT_WHEN_EMPTY를 보니
“빈 값을 입력했을 때 0이 아닌 다른 값을 반환해야 한다” 라는 점을 고려하셔서 작성하신 코드였군요! 👍

의도적으로 작성하셨다면 좋습니다~! 😀
저는 혹시나 reduce 메서드의 다른 작동 방식을 공유하고자 리뷰를 드린 부분이었어요~